qutebrowser/misc/utilcmds.py

```diff
@@ -86,6 +86,19 @@ def repeat(times: int, command, win_id):
         commandrunner.run_safely(command)
 
 
+@cmdutils.register(maxsplit=1, hide=True, no_cmd_split=True,
+                   no_replace_variables=True)
+@cmdutils.argument('win_id', win_id=True)
+def run_with_count(times: int, command, win_id):
+    """Run a command with the given count.
+
+    Args:
+        times: The count to pass to the command.
```

Comment:
`times` sounds like a weird name for that argument - I think `count` as argument name shouldn't be hardcoded anywhere anymore, so maybe just use that?